### PR TITLE
Story/9034

### PR DESCRIPTION
**Pull request by samclk**

### CODE REVIEW

File: addons/udes_stock/static/src/xml/create_planned_transfer_button.xml [hunk 1]
```diff
@@ -13,15 +13,9 @@
         </t>
     </t>
 
-    <!--Replace the create button on a picking formview to have 'Planned Transfer' functionality instead of 'Immediate Transfer'-->
+    <!--Remove the create button on a picking form view-->
     <t t-extend="FormView.buttons">
         <t t-jquery="button.o_form_button_create" t-operation="replace">
-            <t t-if="widget and widget.modelName=='stock.picking'">
```
Comment (nocturnalastro) [inline addons/udes_stock/static/src/xml/create_planned_transfer_button.xml]: Don't we want this if the person is an admin or privilaged user?

File: addons/udes_stock/views/edi_menu_views.xml [hunk 2]
```diff
@@ -6,5 +6,9 @@
     <menuitem id="edi_record_menu" name="UDES"
 	      parent="edi.record_menu" sequence="500"/>
 
+    <!-- EDI move tracker menu item -->
```
Comment (nocturnalastro) [inline addons/udes_stock/views/edi_menu_views.xml]: does udes_stock depennd on edi? if not this needs to live somewhere else.

Comment (joe-gb) [reply addons/udes_stock/views/edi_menu_views.xml]: it does!

File: addons/udes_stock/static/src/xml/create_planned_transfer_button.xml (hunk 1 above)
Comment (joe-gb) [reply addons/udes_stock/static/src/xml/create_planned_transfer_button.xml]: don't think it's needed in form view, we still have the "create" on list view, and "new planned transfer" on the picking widgets - I see no use for a "Create" button when already looking at a picking! but it's one we can definitely ask to product :)

**Pull request merged**